packages/website/tests/index.spec.ts

```diff
@@ -15,8 +15,6 @@ test.describe('Website', () => {
       }
     });
     await page.goto('/');
-    expect(errorMessages).toStrictEqual([
-      "[error] Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more: https://reactjs.org/link/switch-to-createroot",
```

Comment:
This error message was only applicable to the local dev-server and is not shown to our users on production builds. A nice extra benefit of this approach is that our testing is more representative of our end UX